Sources/KituraNet/HTTP/HTTPServer.swift

```diff
@@ -276,14 +299,23 @@ public class HTTPServer: Server {
             .childChannelOption(ChannelOptions.socket(IPPROTO_TCP, TCP_NODELAY), value: 1)
 
         do {
-            serverChannel = try bootstrap.bind(host: "0.0.0.0", port: port).wait()
+            if case let SocketType.tcp(port) = socket {
+                serverChannel = try bootstrap.bind(host: "0.0.0.0", port: port).wait()
+            } else if case let SocketType.unix(unixDomainSocketPath) = socket {
+                serverChannel = try bootstrap.bind(unixDomainSocketPath: unixDomainSocketPath).wait()
+            }
             self.port = serverChannel?.localAddress?.port.map { Int($0) }
             self.state = .started
             self.lifecycleListener.performStartCallbacks()
         } catch let error {
             self.state = .failed
             self.lifecycleListener.performFailCallbacks(with: error)
-            Log.error("Error trying to bind to \(port): \(error)")
+            switch socket {
+            case .tcp(let port):
+                Log.error("Error trying to bind to \(port): \(error)")
+            case .unix(let socketPath):
+                Log.error("Error trying to bind to \(socketPath): \(error)")
+            }
             throw error
         }
 
```

Comment:
The line below this one:
```swift
Log.info("Listening on port \(self.port!)")
```
blows up if you listen on a Unix socket and have logging enabled...

Reply:
I pushed a fix for this